### DISCUSSION TITLE
Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by BTOdell** [openvinotoolkit/openvino#23899] [General]
Just curious how quickly OpenVINO will be made available/compatible with Ubuntu 24.04 when it's released on April 25th? Has there been any experiments done against the nightly Ubuntu releases? It seems that using OpenVINO's `ubuntu22 main` apt repo on Ubuntu 24.04 works fine, although runtime compatibility issues worry me.

**Comment by ilya-lavrenov** [comment]
Could you please check https://docs.openvino.ai/2024/get-started/install-openvino/install-openvino-apt.html ?
I suppose debian packages for Ubuntu 22.04 will work on Ubuntu 24.04 as well.

Meanwhile, created a PR for explicit Ubuntu 24.04 support
https://github.com/openvinotoolkit/openvino/pull/23905
